cryptus-app/components/wallet_viewer/nft_details/nft_properties/property_block/property_block.module.scss

```diff
@@ -0,0 +1,35 @@
+@use "/styles/variables" as v;
+
+.container {
+    flex: 1;
+    flex-basis: 50%;
```

Comment:
/* this is shorthand */
  flex: 0 1 auto;

  /* for this */
  flex-grow: 0;
  flex-shrink: 1;
  flex-basis: auto;

so use flex: 1 1 50%